src/modules/expense/Form.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import { styled, Theme } from '@mui/material/styles';
+
+const StyledDiv = styled(`div`)(({ theme }: { theme: Theme }) => ({
+  display: 'flex',
+  width: '100%',
+}));
+
+export const Form = () => {
+  const [category, setCategory] = React.useState('');
+
+  const handleChangeCategory = (event: SelectChangeEvent) => {
+    setCategory(event.target.value as string);
+  };
+
+  return (
+    <StyledDiv sx={{ minWidth: 120 }}>
```

Comment:
This should be a form element. All elements of the form (except image preview) should have 100% width but use flex (or Stack MUI element) to align them instead of using width.